openlibrary/catalog/get_ia.py

```diff
@@ -43,33 +34,25 @@ def bad_ia_xml(ia):
     return '<!--' in urlopen_keep_trying(base + loc).read()
 
 def get_marc_ia_data(ia, host=None, path=None):
-    ia = ia.strip() # 'cyclopdiaofedu00kidd '
+    """
+    DEPRECATED
+    """
     ending = 'meta.mrc'
     if host and path:
         url = 'http://%s%s/%s_%s' % (host, path, ia, ending)
     else:
-        url = 'http://www.archive.org/download/' + ia + '/' + ia + '_' + ending
+        url = base + ia + '/' + ia + '_' + ending
```

Comment:
where's base coming from, can this be capitalized so it's more clear its a constant?

Reply:
This should really be read from the config file, does capitalizing still apply if we read it from config?